core/templates/dev/head/pages/exploration_editor/history_tab/HistoryTab.js

```diff
@@ -259,5 +263,20 @@ oppia.controller('HistoryTab', [
         });
       });
     };
+
+    $scope.computeVersionsToDisplay = function() {
+      var begin = (($scope.currentPage - 1) * $scope.versionsPerPage);
+      var end = Math.min(
+        begin + $scope.versionsPerPage, $scope.versionCheckboxArray.length);
+      $scope.versionNumbersToDisplay = [];
+      for (var i = begin; i < end; i++) {
+        $scope.versionNumbersToDisplay.push(
+          $scope.versionCheckboxArray[i].vnum);
+      }
+    };
+
+    $scope.pageChanged = function() {
```

Comment:
$scope.onPageChanged

But I'm not sure why you need this function at all, you can just call computeVersionsToDisplay directly from the directive.

Reply:
Yup right, don't really need it! Removed.